### ISSUE TITLE
Database key duplication

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#361]
**Reported by valleegr on 1 Aug 2007 14:27 UTC**
We periodically have database key duplication at least when working on OSCAR (which implies we may not go through all the steps all the time):

```
List of available OPKGs:
[ /opt/oscar/packages/apitest /opt/oscar/packages/base /opt/oscar/packages/c3 /opt/oscar/packages/disable-services /opt/oscar/packages/ganglia /opt/oscar/packages/jobmonarch /opt/oscar/packages/lam /opt/oscar/packages/linux-ha /opt/oscar/packages/loghost /opt/oscar/packages/maui /opt/oscar/packages/mpich /opt/oscar/packages/mta-config /opt/oscar/packages/netbootmgr /opt/oscar/packages/networking /opt/oscar/packages/ntpconfig /opt/oscar/packages/oda /opt/oscar/packages/openmpi /opt/oscar/packages/opium /opt/oscar/packages/pfilter /opt/oscar/packages/pvm /opt/oscar/packages/rapt /opt/oscar/packages/sc3 /opt/oscar/packages/sge /opt/oscar/packages/sis /opt/oscar/packages/switcher /opt/oscar/packages/sync_files /opt/oscar/packages/torque /opt/oscar/packages/yume ]
Array: 28 elements
ERROR 1062 (23000) at line 311: Duplicate entry '0' for key 1
DB_DEBUG>/opt/oscar/scripts/package_config_xmls_to_database:
====> cannot create OSCAR tables
at /opt/oscar/scripts/package_config_xmls_to_database line 463.
Couldn't set up oda database at ./wizard_prep line 383
```

**Comment by dikim33** [comment]
**Modified by valleegr on 1 Aug 2007 14:30 UTC**

**Comment by dikim33** [comment]
**Comment by dikim on 13 Sep 2007 05:43 UTC**
(In [6158]) Try to fix #361 "Database key duplication" issue. I hope that this works
fine.
